### DISCUSSION TITLE
Does supporting a new layout require a new/fresh release?

### DISCUSSION THREAD

**Discussion by surendrasinghs** [Sunbird-Saral/Community#3] [General]
In the latest (v1.5.3) release, I read that it supports a few specific layouts. I want to understand what it takes to support a new layout not yet supported? What's the process and how to go about designing a layout?

**Comment by dileep-gadiraju** [comment]
@surendrasinghs, a new layout can be generated and configured in the backend directly if the current App(APK/Bundle) used has all the functional features required already present.  Layout generation and configuration procedure is documented @ https://saral.sunbird.org/use/layout-configuration.
